tardis/montecarlo/tests/test_cmontecarlo.py

```diff
@@ -0,0 +1,47 @@
+import os
+import random
+from ctypes import CDLL, c_double
+
+import pytest
+import numpy as np
+from numpy.testing import assert_almost_equal
+
+from tardis import __path__ as path
+
+test_path = os.path.join(path[0], 'montecarlo', 'montecarlo.so')
+
+tests = CDLL(test_path)
+
+tests.init_rpacket()
+tests.init_storage_model()
+
+def test_rpacket_doppler_factor():
+	doppler_factor = 0.9995413494
+	tests.test_rpacket_doppler_factor.restype = c_double
+	assert_almost_equal(tests.test_rpacket_doppler_factor(), doppler_factor)
+
+def test_compute_distance2boundary():
+	distance_to_boundary = -0.0109999775
+	tests.test_compute_distance2boundary.restype = c_double
+	assert_almost_equal(tests.test_compute_distance2boundary(), 
+		distance_to_boundary)
+
+'''
+def test_compute_distance2line():
+	distance_to_line = 0.0
+	tests.test_compute_distance2line.restype = c_double
+	assert_almost_equal(tests.test_compute_distance2line(),
+		distance_to_line)
+'''
```

Comment:
@wkerzendorf  If I uncomment this function, its throwing an error. Any idea why ? I'll finish the other montecarlo functions similarly by today if this PR is okay with you.